docs/how_to/namespaced_apphooks.rst

```diff
@@ -95,18 +199,22 @@ Next lets define the AppHookConfig model (in ``cms_appconfig.py``):
         title = forms.CharField()
     setup_config(FaqConfigForm, FaqConfig)
 
-The implementation can be completely empty as the minimal schema is defined in
-the parent (abstract) model.
+The implementation *can* be left completely empty, as the minimal schema is already defined in
+the abstract parent model provided by Aldryn Apphooks Config.
+
+Here though we're defining an extra field on model, ``paginate_by``. We'll use it later
+to control how many entries should be displayed per page.
+
+We also set up a ``FaqConfigForm``, which uses ``AppDataForm`` to add a field to ``FaqConfig`` without actually
+touching its model. **What is the actual point of this?**
```

Comment:
``FaqConfigForm`` is the form that handles all the schemaless fields defined on the apphook config. It's just a convenient way to add more options without generating migrations